packages/harmony/src/components/hover-card/HoverCardHeader.tsx

```diff
@@ -36,18 +34,15 @@ export const HoverCardHeader = ({
           {title}
         </Text>
       </Flex>
-
-      {onClose ? (
+      {iconRight ? (
```

Comment:
so is the move to just use iconbutton unconditionally even if there is no on-close?

Reply:
i think the condition is if there's an icon passed it'll show the icon and if there's an onClose passed it'll use that too. probably safe to assume any onClose should have an icon